network/endpoint.go

```diff
@@ -55,6 +56,7 @@ type EndpointInfo struct {
 	MacAddress               net.HardwareAddr
 	DNS                      DNSInfo
 	IPAddresses              []net.IPNet
+	IPsToRouteViaHost        []string
```

Comment:
@tamilmani1989 If I remove this line, make all-binaries fails saying:

> epInfo.IPsToRouteViaHost undefined (type *EndpointInfo has no field or method IPsToRouteViaHost)

Reply:
dont remove it from endpointinfo..just remove from endpoint